exercises/practice/react/react.go

```diff
@@ -1 +1,35 @@
 package react
+
+// Define reactor, cell and canceler types here.
+
```

Comment:
```suggestion
// Define reactor, cell and canceler types here.
// These types will implement the Reactor, Cell and Canceler interfaces, respectively.

```
I think this bit of clarification is important.